src/base/projects/Widget.svelte

```diff
@@ -81,37 +106,62 @@
   }
 </style>
 
-<article on:click class:project-faded={faded}>
-  <div class="id">
-    <span class="name">{project.name}</span>
-    <span class="urn desktop">{project.urn}</span>
-  </div>
-  <div class="description">{project.description || ""}</div>
-  <div class="anchor">
-    <span class="commit">
-      <slot name="stateHash">
-        {#if project.head}
-          {project.head}
-        {:else}
-          <span class="subtle">✗ No head</span>
-        {/if}
-      </slot>
-    </span>
-    <span class="anchor-info">
-      <span class="actions">
-        <slot name="actions">
-        </slot>
-      </span>
-      <span class="anchor-badge">
-        <slot name="anchor">
-          {#if anchor && project.head}
-            <AnchorBadge
-              commit={project.head}
-              head={project.head} noText noBg
-              anchors={[anchor.anchor.stateHash]} />
+<article
+  on:click
+  on:mouseover={() => widgetHover = true}
+  on:focus={() => widgetHover = true}
+  on:mouseout={() => widgetHover = false}
+  on:blur={() => widgetHover = false}
+  class:project-faded={faded}>
+  <div class="left">
+    <div class="id">
+      <span class="name">{project.name}</span>
+    </div>
+    <div class="description">{project.description || ""}</div>
+    <div class="anchor">
+      <span class="commit">
+        <slot name="stateHash">
+          {#if project.head}
+            {project.head}
+          {:else}
+            <span class="subtle">✗ No head</span>
           {/if}
         </slot>
       </span>
-    </span>
+    </div>
+  </div>
+  <div class="right">
+    <div class="id">
+      <span class="urn desktop">{project.urn}</span>
+    </div>
+    {#if widgetHover}
```

Comment:
Can CSS not be used for this? `article:hover .anchor { display: block }` `article:hover .activity { display: none }`?